manifests/charts/istiod-remote/files/gen-istiod-remote.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: istio-sidecar-injector
-  namespace: default
+  namespace: opentlc-mgr-smcp
```

Comment:
why is this change a part of this PR ?

Reply:
This change came from my environment. I backed it out. That said, the change wasn't in my environment variables or anywhere else in the codebase other than the output of `make gen`. I need to dig in to see why this is occurring.